app/helpers/time.rb

```diff
@@ -0,0 +1,21 @@
+
```

Comment:
A) your indentation is off. Don't be sloppy with your code formatting!
B) `prettify` is an ambiguous name. Be more specific. Something like `human_readable_date(date)`?
C) All helpers need to be wrapped in a `helpers` block